march_rqt_note_taker/src/march_rqt_note_taker/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .entry import Entry
+from .entry_model import EntryModel
+from .filter_map import FilterMap
+from .notes_plugin import NotesPlugin
+from .notes_widget import NotesWidget
```

Comment:
Can this be moved to the __init__ in the test folder?

Reply:
I think so, let me try

Reply:
It worked :+1: